candidate-testing.js

```diff
@@ -1,35 +1,42 @@
-const input = require('readline-sync');
-
 // TODO 2: modify your quiz app to ask 5 questions //
 
 // TODO 1.1a: Define candidateName // 
-let candidateName;
+let candidateName = "";
 // TODO 1.2a: Define question, correctAnswer, and candidateAnswer //
-let question;
-let correctAnswer;
-let candidateAnswer;
+let question = "";
+let correctAnswer = "";
+let candidateAnswer = "";
 let questions;
 let correctAnswers;
 let candidateAnswers;
-
+let space = " ";
 
 function askForName() {
   // TODO 1.1b: Ask for candidate's name //
+const input = require('readline-sync');
+ candidateName = input.question("Please enter your name: " );
 
+console.log("Hello," + space + candidateName +"!");
 }
 
 function askQuestion() {
   // TODO 1.2b: Ask candidate the question and assign the response as candidateAnswer //
-
-
+const input = require('readline-sync');
+question = input.question("Who was the first American woman in space? ")
+candidateAnswer = question;
```

Comment:
I see what you did here on lines 25 and 26.... if we declare and initialize the question on line 6(see my comment on line 6), is there a way to combine lines 25 and 26 into one?